spec/services/doc_auth/socure/request_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe DocAuth::Socure::Request do
+  subject(:request) { described_class.new }
+
+  describe 'a new request' do
+    it 'exists' do
+      expect(request).to be
+    end
+  end
+
+  describe '#fetch' do
```

Comment:
should we also have a test for:
# if http_response.success? != true
# a successful request where where http_response.success? == true